tests/basic/expected_stdxxx_encoding.py

```diff
@@ -0,0 +1,28 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+import sys
+import subprocess
+
+# For various OSes the encoding is different.
+if sys.platform.startswith('win'):
+    # The default Windows encoding is based on the active console code page,
+    # which varies from PC to PC and may be changed by the user. For more info,
+    # See https://technet.microsoft.com/en-us/library/bb490874.aspx.
+    # Therefore, ask for the current encoding using chcp (see link above
+    # for details). The expected output: 'Active code page: nnn\r\n'.
+    # We want only the nnn (a number which specifies the code page).
+    chcp_out = subprocess.check_output(['chcp'], shell=True)
```

Comment:
@codewarrior0 is correct -- `chcp` is a shell builtin, so it won't run without `shell=True`. However, I also agree with David that the patch is weird: it's comparing Pyinstaller to hand-generated results. It should compare CPython output to Pyinstaller output: create a `.pickle` in CPython (probably in `runtests.py`), then load that in the Pyinstaller executable and perform a series of asserts. Future work...